articles/confidential-computing/enclave-aware-containers.md

```diff
@@ -47,10 +47,9 @@ This solution allows you to bring existing ML trained model and run them confide
 
 Get started with ML model lift and shift to ONNX runtime [here](https://aka.ms/confidentialinference)
 
-### Edgeless RT
-
-Edgeless RT is an open-source project that builds upon the Open Enclave SDK. It adds support for Go and additional C++ features. Get started with a simple confidential Go application using your familiar VS Code environment [here](https://github.com/edgelesssys/edgelessrt). For Edgeless applications on AKS follow instructions [here](https://github.com/edgelesssys/edgelessrt/blob/master/docs/ERTAzureAKSDeployment.md)
+### EGo
 
+The open source [EGo SDK](https://www.ego.dev) brings support for the Go programming language to enclaves. EGo builds upon the Open Enclave SDK and aims at making it easy to bring cloud-native services to the enclave world. For instance, if your goal is to make a Go-based microservice deployment on AKS confidential, then EGo can be a starting point.
```

Comment:
Please update to "The open source [EGo SDK](https://www.ego.dev) brings support for the Go programming language to enclaves. EGo builds upon the Open Enclave SDK and aims at making it easy to bring cloud-native services to the enclave world. EGo is a built to make a Go-based microservice deployment on AKS confidential, Get started with by deploying a sample referenced [here](https://github.com/edgelesssys/edgelessrt/blob/master/docs/ERTAzureAKSDeployment.md)

Reply:
Thanks for reviewing Amar! Will do.